main.go

```diff
@@ -186,200 +296,230 @@ func newCreateCustomerCommand(db *sql.DB) *cli.Command {
 			if err != nil {
 				return err
 			}
-			w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
-			fmt.Fprintln(w, "Customer_id\tEmail\tState\t")
-			fmt.Fprintf(w, "%d\t%s\t%s\t\n", customerID, email, state)
-			w.Flush()
+			printCustomer(os.Stdout, &Customer{int(customerID), email, state})
 
 			return nil
 		},
 	}
 }
 
+func newCreateProductCommand(db *sql.DB) *cli.Command {
+	return &cli.Command{
+		Name:  "create-product",
+		Usage: "Creates a new product to go in the products database, must specify name and price",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "sku",
+				Usage: "the sku of the product",
+			},
+		},
+		Action: func(cCtx *cli.Context) error {
+			if cCtx.NArg() < 2 {
+				return errors.New("Must specify name and price")
+			}
+
+			name := cCtx.Args().Get(0)
+			price, err := strconv.ParseFloat(cCtx.Args().Get(1), 32)
+			price = (price * 100) / 100
```

Comment:
I'm not sure what this is doing?

Reply:
was trying something out to fix the float printing issue and forgot to remove it